features/FEATURE_BLE/targets/TARGET_STM/TARGET_STM32WB/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(mbed-os
+    PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${CMAKE_CURRENT_SOURCE_DIR}/STM32Cube_FW
```

Comment:
one question - we use here this current source but not below in target_sources, why?

Reply:
You mean the usage of `${CMAKE_CURRENT_SOURCE_DIR}`?
When I started working on CMake, I noticed the usage of `.` in include directories and thought we should just use the provided CMake variable. Target sources were never listed as `./filename.ext` (there were just listed as `filename.txt`) so I never replaced had to replace the `.` in target sources.

Since then, I have been consistent in only using `${CMAKE_CURRENT_SOURCE_DIR}` for include dirs. We can use it for both.